packaging/patches/use-snapctl-with-slot-cups-server.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/scheduler/auth.c b/scheduler/auth.c
+index 7f6c3f6d2..326d70832 100644
+--- a/scheduler/auth.c
++++ b/scheduler/auth.c
+@@ -2006,7 +2006,7 @@ check_admin_access(cupsd_client_t *con) // I - Client connection
+     "is-connected",
+     "--apparmor-label",
+     NULL,
+-    "cups-control",
```

Comment:
Is this a generic change that should always be applied? If so, apply it and don't patch around issues in the "upstream" code (now that this will be upstream).

If we need this to be configurable, make it configurable.

Reply:
Yes, there should not be any patches in a repo which patch anything in the same repo. Whatever gets patched here has to be done right in the first place, or be configurable if the patch is only applied for special cases.

This patch is always applied when we build the Snap and the code it patches is only relevant for the snapped CUPS, so, @rudra-iitm , here you should directly modify the file `scheduler/auth.c` to use `cups-server` instead of `cups-control`.

Please also check with other patches should you have moved some into the CUPS repo.